app/assets/stylesheets/styles/components/users/_profile.css.scss

```diff
@@ -0,0 +1,136 @@
+.member-profile {
+
+  ul {
+    list-style: none;
+    margin-left: 0;
+  }
+
+  .member-content {
+    margin-top: 2rem;
+  }
+
+  .edit-profile-button {
+    margin-bottom: 1rem;
+    margin-right: 1rem;
+  }
+
+  .create-guide-button a {
+    color: #ffffff;
+  }
+
+  .profile-sub-header {
+    ul {
+      text-align: center;
+    }
+
+    li {
+      display: inline-block;
+      padding: 0.4rem 1rem;
+      margin: 0.4rem 0;
+      background-color: #e9e9e9;
+    }
+
+    .number {
+      font-weight: bold;
+      font-size: 1.2rem;
+      margin: 0;
+    }
+
+    .attribute {
+      color: #999999;
+      margin-bottom: 0.4rem;
+      font-weight: bold;
+    }
+  }
+
+  .sidebar-section, .section {
+    margin-bottom: 3rem;
+  }
+
+  .section-title {
+    display: block;
+    float: left;
+  }
+
+  .section-header {
+    border-bottom: 2px solid #cecece;
+    display: inline-block;
+    width: 100%;
+  }
+
+  .guides-list {
+
+    margin-top: -0.4rem;
+
+    table {
+      border: none;
+      border-collapse: collapse;
+    }
+
+    .compatibility-circle {
+      float: left;
+      margin: 0.2rem 1rem;
+    }
+
+    .guide-name {
+      color: #333333;
+      font-weight: bold;
+      font-size: 1.3rem;
+      line-height: 56px;
+    }
+
+    .crop-name {
+      color: #666666;
+      font-style: italic;
+      font-size: 1rem;
+    }
+  }
+
+  .member-info-wrapper {
+
+    padding-top: 5rem;
+    padding-bottom: 5rem;
+
+    @media #{$small-only} {
+      padding-top: 2rem;
+      padding-bottom: 2rem;
+    }
+
+    .member-picture {
+      background-size: cover;
+      background-repeat: no-repeat;
+      background-position: center;
+      border-radius: 70px;
+      border: 3px solid #ffffff;
+      float: right;
+      margin-right: 1rem;
+      height: 140px;
+      width: 140px;
+
+      @media #{$small-only} {
+        float: none;
+        margin-left: auto;
+        margin-right: auto;
+        margin-bottom: 1rem;
+        height: 100px;
+        width: 100px;
+      }
+    }
+
+    .member-infos {
+
+      h2 {
+      font-weight: bold;
+      }
+
+      h2, h5 {
+        color: #ffffff;
+        text-shadow: 1px 1px 20px #666666;
```

Comment:
Color literals like `#666666` should only be used in variable declarations; they should be referred to via variable everywhere else.